instance/models/server.py

```diff
@@ -452,7 +451,7 @@ def terminate(self):
                 self.logger.warning("Server has not reached SHUTOFF state after max wait time; terminating forcefully.")
             os_server.delete()
         except novaclient.exceptions.NotFound:
-            self.logger.error('Error while attempting to terminate server: could not find OS server')
+            self.logger.warning('Error while attempting to terminate server: could not find OS server')
```

Comment:
@kewne, nit: Imho, this should be an error message as this is an error condition. We just don't want to abort due to this error. 😛

Reply:
For me, that's precisely what distinguishes logging something as an error vs logging a warning 😃 

The classic example here is what happens if there is a temporary network issue connecting to the database: the server is deleted from OpenStack but not marked as terminated.
The natural thing, then, would be to retry the operation, which would raise the not found exception; IMO it's incorrect to then log an error, as it incorrectly leads the user to think there is one.

Would you agree with slightly rewording the message to make this clearer, then?
Something like `Failed to terminate server: could not find OS server. It may have been deleted by a previous attempt to terminate.`?

Reply:
>  The natural thing, then, would be to retry the operation, which would raise the not found exception; IMO it's incorrect to then log an error, as it incorrectly leads the user to think there is one.

@kewne, there is an error here since Ocim thinks that there is an OpenStack server and tries to delete it. But there is none. Unless Ocim then cleans up its internal state to reflect that the server no longer exists, this should be an error message imho.